src/seedu/addressbook/data/person/Email.java

```diff
@@ -11,7 +11,8 @@
     public static final String EXAMPLE = "valid@e.mail";
     public static final String MESSAGE_EMAIL_CONSTRAINTS =
             "Person emails should be 2 alphanumeric/period strings separated by '@'";
-    public static final String EMAIL_VALIDATION_REGEX = "[\\w\\.]+@[\\w\\.]+";
+    public static final String EMAIL_VALIDATION_REGEX =
```

Comment:
The [coding standard](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit) suggests to have a column limit of 100 characters. Since this line falls within the limit, it is not necessary to separate the lines.